app/dao/services_dao.py

```diff
@@ -461,17 +466,25 @@ def fetch_service_email_limit(service_id: uuid.UUID) -> int:
 
 def fetch_todays_total_email_count(service_id: uuid.UUID) -> int:
     midnight = get_midnight(datetime.now(tz=pytz.utc))
+    scheduled = (
+        db.session.query(func.coalesce(func.sum(Job.notification_count), 0).label("total_scheduled_notifications")).filter(
```

Comment:
Maybe better to not user `coalesce` here to reduce DB load since we can do this kind of processing faster in Python. Thoughts?

Reply:
Doesn't `coalesce()` just check for null and return 0 if the other param is null? Would this be a performance problem?